ui/apps/platform/cypress/integration/vulnmanagement/dashboard.test.js

```diff
@@ -1,122 +1,188 @@
-    cy.get(selectors.getWidget(`Top risky ${entityName} by CVE count & CVSS score`))
-        .find(selectors.viewAllButton)
-        .click();
-    cy.wait('@entities');
-    cy.location('pathname').should('eq', url.list[entityName]);
+
+    const widgetSelector = selectors.getWidget(
+        `Top risky ${entitiesKey} by CVE count & CVSS score`
+    );
+    const widgetViewAllSelector = `${widgetSelector} ${selectors.viewAllButton}`;
+    interactAndWaitForVulnerabilityManagementEntities(() => {
+        cy.get(widgetViewAllSelector).click();
+    }, entitiesKey);
 }
 
-describe('Vuln Management Dashboard Page', () => {
+describe('Vulnerability Management Dashboard', () => {
     withAuth();
 
     it('should show same number of policies between the tile and the policies list', () => {
         visitVulnerabilityManagementDashboard();
-        cy.get(selectors.tileLinks)
-            .eq(0)
-            .find(selectors.tileLinkValue)
+
+        const entitiesKey = 'policies';
+        const tileLinkSelector = `${selectors.tileLinks}:eq(0)`;
+        cy.get(`${tileLinkSelector} ${selectors.tileLinkValue}`)
             .invoke('text')
             .then((value) => {
-                const numPolicies = value;
-                cy.get(selectors.tileLinks).eq(0).click();
-                cy.get(`[data-testid="panel"] [data-testid="panel-header"]`)
-                    .invoke('text')
-                    .then((panelHeaderText) => {
-                        expect(parseInt(panelHeaderText, 10)).to.equal(parseInt(numPolicies, 10));
-                    });
+                interactAndWaitForVulnerabilityManagementEntities(() => {
+                    cy.get(tileLinkSelector).click();
+                }, entitiesKey);
+
+                cy.get(`[data-testid="panel"] [data-testid="panel-header"]:contains("${value}")`);
             });
     });
 
-    // TODO: update CVE links to CVE tables checks, for VM Updates
-    it.skip('should show same number of cves between the tile and the cves list', () => {
-        visitVulnerabilityManagementDashboard();
-        cy.get(selectors.tileLinks)
-            .eq(1)
-            .find(selectors.tileLinkValue)
-            .invoke('text')
-            .then((value) => {
-                const numCves = value;
-                cy.get(selectors.tileLinks).eq(1).click();
-                cy.get(`[data-testid="panel"] [data-testid="panel-header"]`)
-                    .invoke('text')
-                    .then((panelHeaderText) => {
-                        expect(parseInt(panelHeaderText, 10)).to.equal(parseInt(numCves, 10));
-                    });
-            });
+    // TODO Delete skip when we delete ROX_FRONTEND_VM_UPDATES feature flag.
+
+    it.skip('should show same number of Image CVEs in menu item and entities list', () => {
+        verifyVulnerabilityManagementDashboardCVEs('image-cves', /^\d+ Image CVEs?$/);
+    });
```

Comment:
You abstracted the setup into a function `verifyVulnerabilityManagementDashboardCVEs()`, but did you mean to leave all three of these tests that now use it `skip`'ed ?

Reply:
Good eyes. I tested them locally and decided on temporary skip instead of feature flag logic.

When we turn on the feature flag for nightly and release builds, we can remove the skip.